src/Modules/score.js

```diff
@@ -1,9 +1,26 @@
-const { v4: uuidv4 } = require('uuid');
+// const { v4: uuidv4 } = require('uuid');
 
-export default class PIECE {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
-    this.id = uuidv4();
-  }
-}
\ No newline at end of file
+// export default class PIECE {
+//   constructor(name, score) {
+//     this.name = name;
+//     this.score = score;
+//     this.id = uuidv4();
+//   }
+// }
+// 'Game with ID: GrlP35gyONINxwNnhDtc added.'}
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented lines of code in order for us to have a cleaner code base.